frontend_tests/node_tests/list_widget.js

```diff
@@ -6,15 +6,17 @@ const {mock_cjs, mock_esm, set_global, zrequire} = require("../zjsunit/namespace
 const {run_test} = require("../zjsunit/test");
 const blueslip = require("../zjsunit/zblueslip");
 
+const sb = mock_esm("../../static/js/sb", {
+    get_scroll_element: (element) => element,
+});
```

Comment:
Would `scrollbar_ui.get_scroll_element` be a reasonable name?  We don't use abbreviations like `sb` for any other module names, and I think that's a pretty reasonable name for this bundle of functions.

Reply:
I prefer the short name here. If you use a longer name you get some weird stuff from the prettier tool for some of the longer lines.

Reply:
My guess is those are probably all things where we should actually just write clearer code, e.g. having a `$scroll_element` variable rather than chaining things.

Reply:
There’s no need to abbreviate module names just to avoid writing `scrollbar_ui.get_scroll_element`. We could instead just switch the wildcard import to a named import:

```js
import {get_scrollbar_element} from "./scrollbar_ui";

get_scroll_element(…)
```